openeo_fastapi/client/models.py

```diff
@@ -0,0 +1,604 @@
+import sys
```

Comment:
@SerRichard @GeraldIr Do we want to declare all the STAC related models here? or import them from stac-utils / stac-fastapi ?

Reply:
Just took a quick look, unless I'm missing something, they don't seem to have a number of the models? But, if I've missed something and a bunch of these can be imported and not defined here, sure thing!

Reply:
ok, will import what has already been definded in stac-utils and keep the rest